Meta/CMake/download_icc_profiles.cmake

```diff
@@ -0,0 +1,21 @@
+include(${CMAKE_CURRENT_LIST_DIR}/utils.cmake)
+
+if (ENABLE_ADOBE_ICC_PROFILES_DOWNLOAD)
+    set(ADOBE_ICC_PROFILES_PATH "${SERENITY_CACHE_DIR}/AdobeICCProfiles" CACHE PATH "Download location for Adobe ICC profiles")
+    set(ADOBE_ICC_PROFILES_DATA_URL "https://download.adobe.com/pub/adobe/iccprofiles/win/AdobeICCProfilesCS4Win_end-user.zip")
+    set(ADOBE_ICC_PROFILES_ZIP_PATH "${ADOBE_ICC_PROFILES_PATH}/adobe-icc-profiles.zip")
+    download_file("${ADOBE_ICC_PROFILES_DATA_URL}" "${ADOBE_ICC_PROFILES_ZIP_PATH}")
```

Comment:
this call to download_file should be guarded by `if (ENABLE_NETWORK_DOWNLOADS)` like the other calls. It's my attempt to allow offline builds. Perhaps you have some improvements in that department as well :)

Reply:
Done.

Do you have any particular improvements in mind?

(Maybe download_file() could contain the check for ENABLE_NETWORK_DOWNLOADS? Also, seems weird that the other files also guard the unzipping behind ENABLE_NETWORK_DOWNLOADS – not doing that for this one.)